dot_lpdev_cmds.sh

```diff
@@ -682,12 +694,60 @@ function __lpdev_node_transcoder {
   echo "Requesting test tokens"
   curl -X "POST" http://localhost:$transcoderApiPort/requestTokens
 
+  echo "Initializing current round"
+  curl -X "POST" http://localhost:$transcoderApiPort/initializeRound
+
   echo "Activating transcoder"
-  curl -X "POST" http://localhost:$transcoderApiPort/activateTranscoder\
-    --data-urlencode "blockRewardCut=10&feeShare=5&pricePerSegment=1&amount=500"
+  curl -d "blockRewardCut=10&feeShare=5&pricePerSegment=1&amount=500" \
+    -H "Content-Type: application/x-www-form-urlencoded" \
+    -X "POST" http://localhost:$transcoderApiPort/activateTranscoder\
 
 }
 
+function __lpdev_verifier_init {
+  if [ -d $srcDir/verification-computation-solver ]
+  then
+    echo "Protocol src directory exists"
+  else
+    echo "Cloning github.com/livepeer/verification-computation-solver into src directory"
+    OPWD=$PWD
+    cd $srcDir
+    git clone "https://github.com/livepeer/verification-computation-solver.git"
+    cd $OPWD
+  fi
+
+  ##
+  # Update npm
+  ##
+
+  listModules=($(ls $srcDir/verification-computation-solver/node_modules))
+  if [ -d $srcDir/verification-computation-solver/node_modules ] && [ ${#listModules[@]} -gt 0 ]
+  then
+    echo "Npm packages already installed"
+  else
+    echo "Running \`npm install\`"
+    OPWD=$PWD
+    cd $srcDir/protocol
```

Comment:
A few lines up, it checks for the presence of node_modules in `$srcDir/verification-computation-solver/node_modules` . Otherwise, it does a `npm install` in `$srcDir/protocol` . Aren't they different locations?

Maybe that's why I'm getting this error when trying to run the verifier: `Error: Cannot find module 'web3'` : the web3 module exists in `protocol/node_modules` but not `verification-computation-solver/node_modules` .  I'm not certain that Node is looking in the correct place for the web3 module.

Reply:
Yep you are right.